tutorials/interoperability/xcm-transfers/from-relaychain-to-parachain.md

```diff
@@ -0,0 +1,108 @@
+---
+title: From Relay Chain to Parachain
+description: Learn how to perform a reserve-backed asset transfer between a relay chain and a parachain using XCM for cross-chain interoperability.
+---
+
+# From Relay Chain to Parachain
+
+## Introduction
+
+[Cross-Consensus Messaging (XCM)](/develop/interoperability/intro-to-xcm/){target=\_blank} facilitates asset transfers both within the same consensus system and between different ones, such as between a relay chain and its parachains. For cross-system transfers, two main methods are available:
+
+- **Asset teleportation** - a simple and efficient method involving only the source and destination chains, ideal for systems with a high level of trust
+- **Reserve-Backed transfers** - involves a trusted reserve holding real assets and mints derivative tokens to track ownership. This method is suited for systems with lower trust levels
+
+In this tutorial, you will learn how to perform a reserve-backed transfer between a relay chain (Polkadot) and a parachain (Astar).
+
+## Prerequisites
+
+To send messages between different consensus systems, you must first open HRMP channels. For detailed guidance, refer to the [XCM Channels](/develop/interoperability/xcm-channels/#xcm-channels){target=\_blank} article before for further information about.
+
+This tutorial uses Chopsticks to fork a relay chain and a parachain connected via HRMP channels. For more details on this setup, see the [XCM Testing](/tutorials/polkadot-sdk/testing/fork-live-chains/#xcm-testing){target=\_blank} section on the Chopsticks page.
+
+## Setup
+
+To simulate XCM operations between different consensus systems, start by forking the network with the following command:
+
+```bash
+chopsticks xcm -r polkadot -p astar
+```
+After executing this command, the relay chain and parachain will expose the following WebSocket endpoints:
+
+=== "Polkadot (relay chain)"
```

Comment:
I think a list item will be better here

Reply:
I tried this before, but I wasn't super convinced, tbh. I think this way the endpoints are displayed in a really easy way to copy and paste, and also it's a bit shorter

Maybe we can leave this decision to the formatting team

Reply:
Ah yeah, I like the copy/paste aspect. I agree about the list, it's not great for this type of thing. We encountered this same issue with the Networks page. Another option would be to do a table:

```
| Chain                  | WebSocket Endpoint                   |
|------------------------|--------------------------------------|
| Polkadot (relay chain) | <pre>```ws://localhost:8001```</pre> |
| Astar (parachain)      | <pre>```ws://localhost:8000```</pre> |
```

<img width="1083" alt="Screenshot 2024-12-04 at 1 35 18 AM" src="https://github.com/user-attachments/assets/3016b397-e501-4d80-b0d8-11bbdd3b02e5">

But I'm not opposed to the tabs in this case